tests/core/Jalasoft.TeamUp.Projects.Core.Tests/InvitationsServiceTests.cs

```diff
@@ -47,7 +48,7 @@ public static IEnumerable<Invitation> MockInvitations()
                     TextInvitation = "We invite you to collaborate with the development team",
                     StartDate = DateTime.Today.AddDays(-10),
                     ExpireDate = DateTime.Today.AddDays(+10),
-                    Status = "invited"
+                    Status = "Invited"
```

Comment:
please create a dictionary in the models that includes all the statuses so we can use it in this kind of situations, this way the developer will know that type of the statuses we have and wouldn't need to determine the write the exact name (this can easily cause mistakes)

Reply:
I'd suggest an enumerable